src/database/all.sql

```diff
@@ -266,51 +323,59 @@ COMMIT;
   
 -- ---------------------------- 
 -- From: 320_insert_service.sql 
```

Comment:
Hey Carl, please make sure that the below fields are in the database, keeping exactly to what is written below. I have no issues if you wanted to included others but please make sure the below are inserted exactly as they are listed in this comment.

Tree Removal
Roof Cleaning
Fence Installation
Plumbing
Oven Repairs